js/src/modal.js

```diff
@@ -245,7 +245,6 @@ class Modal extends BaseComponent {
     this._isTransitioning = false
 
     this._backdrop.hide(() => {
-      document.body.classList.remove(CLASS_NAME_OPEN)
```

Comment:
do not tdelete this. Just put it on line 246

Reply:
Done